src/Onboarding/OnboardingScreen.tsx

```diff
@@ -187,6 +185,11 @@ const style = StyleSheet.create({
     flex: 1,
     backgroundColor: Colors.secondary10,
   },
+  header: {
+    flex: 0.1,
+    flexDirection: "row",
+    justifyContent: "space-between",
+  },
   languageButtonContainer: {
     ...Outlines.ovalBorder,
     position: "absolute",
```

Comment:
I think we should avoid `absolute` positioning this will break the styles across the devices. Maybe we should open PR to fix this?